.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -2,991 +2,931 @@ name: $(Date:yyyyMMdd)$(Rev:.r)_$(Build.SourceBranchName)_$(BuildID)
-  type: boolean
-  default: true
-- name: buildMarinerV2gen2kata
-  displayName: Build MarinerV2 Gen2 Kata
-  type: boolean
-  default: true
-- name: buildAzureLinuxV2gen2kata
-  displayName: Build AzureLinuxV2 Gen2 Kata
-  type: boolean
-  default: true
-- name: buildMarinerV2ARM64
-  displayName: Build MarinerV2 Gen2 - ARM64
-  type: boolean
-  default: true
-- name: buildAzureLinuxV2ARM64
-  displayName: Build AzureLinuxV2 Gen2 - ARM64
-  type: boolean
-  default: true
-- name: buildMarinerV2gen2TrustedLaunch
-  displayName: Build MarinerV2 Gen2 - Trusted Launch
-  type: boolean
-  default: true
-- name: buildAzureLinuxV2gen2TrustedLaunch
-  displayName: Build AzureLinuxV2 Gen2 - Trusted Launch
-  type: boolean
-  default: true
-- name: buildMarinerV2gen2kataTrustedLaunch
-  displayName: Build MarinerV2 Gen2 Kata - Trusted Launch
-  type: boolean
-  default: false
-- name: buildAzureLinuxV2gen2kataTrustedLaunch
-  displayName: Build AzureLinuxV2 Gen2 Kata - Trusted Launch
-  type: boolean
-  default: false
-- name: build1804fipscontainerd
-  displayName: Build 1804 FIPS containerd
-  type: boolean
-  default: true
-- name: build1804fipsgen2containerd
-  displayName: Build 1804 FIPS Gen2 containerd
-  type: boolean
-  default: true
-- name: build2004fipscontainerd
-  displayName: Build 2004 FIPS containerd
-  type: boolean
-  default: true
-- name: build2004fipsgen2containerd
-  displayName: Build 2004 FIPS Gen2 containerd
-  type: boolean
-  default: true
-- name: build2204fipscontainerd
-  displayName: Build 2204 FIPS containerd
-  type: boolean
-  default: false
-- name: build2204fipsgen2containerd
-  displayName: Build 2204 FIPS Gen2 containerd
-  type: boolean
-  default: false
-- name: build2204arm64gen2containerd
-  displayName: Build 2204 ARM64 Gen2 containerd
-  type: boolean
-  default: true
-- name: build2404arm64gen2containerd
-  displayName: Build 2404 ARM64 Gen2 containerd
-  type: boolean
-  default: false
-- name: build2204containerd
-  displayName: Build 2204 Gen1 Containerd
-  type: boolean
-  default: true
-- name: build2204gen2containerd
-  displayName: Build 2204 Gen2 Containerd
-  type: boolean
-  default: true
-- name: build2404containerd
-  displayName: Build 2404 Gen1 Containerd
-  type: boolean
-  default: false
-- name: build2404gen2containerd
-  displayName: Build 2404 Gen2 Containerd
-  type: boolean
-  default: false
-- name: build2004cvmgen2containerd
-  displayName: Build 2004 CVM Gen2 Containerd
-  type: boolean
-  default: true
-- name: build2204tlgen2containerd
-  displayName: Build 2204 TL Gen2 Containerd
-  type: boolean
-  default: true
-- name: build2204minimalgen1containerd
-  displayName: Build 2204 Minimal Gen1 Containerd
-  type: boolean
-  default: false
-- name: build2204minimalgen2containerd
-  displayName: Build 2204 Minimal Gen2 Containerd
-  type: boolean
-  default: false
-- name: dryrun
-  displayName: Dry run
-  type: boolean
-  default: False
+  - name: build1804containerd
```

Comment:
do we want all this extra identation?

Reply:
on `parameters` it's probably a personal preference. Extra indentation is less confusing for me personally.

`stages` has changed. Prior there was multiple stages with 1 job each. Now here is a single stage with multiple jobs in it.